<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resta1</class>
 <widget class="QMainWindow" name="Resta1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resta1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Peca" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuModo">
    <property name="title">
     <string>Modo</string>
    </property>
    <addaction name="actionTradicional"/>
    <addaction name="actionCruz"/>
    <addaction name="actionMais"/>
    <addaction name="actionBanquinho"/>
    <addaction name="actionFlecha"/>
    <addaction name="actionPiramide"/>
    <addaction name="actionLosango"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuModo"/>
   <addaction name="menuSobre"/>
  </widget>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionTradicional">
   <property name="text">
    <string>Tradicional</string>
   </property>
  </action>
  <action name="actionPiramide">
   <property name="text">
    <string>Piramide</string>
   </property>
  </action>
  <action name="actionCruz">
   <property name="text">
    <string>Cruz</string>
   </property>
  </action>
  <action name="actionMais">
   <property name="text">
    <string>Mais</string>
   </property>
  </action>
  <action name="actionBanquinho">
   <property name="text">
    <string>Banquinho</string>
   </property>
  </action>
  <action name="actionFlecha">
   <property name="text">
    <string>Flecha</string>
   </property>
  </action>
  <action name="actionLosango">
   <property name="text">
    <string>Losango</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Peca</class>
   <extends>QPushButton</extends>
   <header location="global">Peca.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
